src/containers/ConfirmSignup.js

```diff
@@ -6,7 +6,7 @@ import ExpandArea from "../components/ExpandArea";
 import './ConfirmSignup.css'
 
 export default function ConfirmSignup(props) {
-  const { token, username } = props.location.state
+  const { token, username ,key } = props.location.state
```

Comment:
Lets call it as keyTypes instead of key, to make it clear what is it, does it make sense ?

Reply:
yeah sure will do that